frontend/src/layouts/App/styles.ts

```diff
@@ -56,15 +60,31 @@ const Header = styled.header`
 
   .header-info {
     justify-content: flex-start;
-    max-width: 45%;
+
+    ${media.max('tablet')`
+      margin-bottom: 0.5rem;
+    `}
+
+    p.title {
```

Comment:
When posible, use single class, don't know if it's necessary to concatenate tag element + class name